lib/compiler/test/beam_doc_SUITE.erl

```diff
@@ -142,7 +142,7 @@ singleton_docformat(Conf) ->
              otp_doc_vsn => {1,0,0},
              since => "1.0"},
     Doc = #{<<"en">> => <<"Doc test module\n\nMore info here">>},
-    FunMeta = #{ authors => [<<"Beep Bop">>], equiv => <<"main/3">> },
+    FunMeta = #{ authors => [<<"Beep Bop">>], equiv => <<"main/3">>, since => "1.0" },
```

Comment:
This works. Not sure if it should be better in the `singleton_meta` test, since the focus is on the metadata there, even if the tests are pretty similar :)